Tasks/ServiceFabricDeploy/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -50,6 +50,7 @@
   "loc.messages.PublishProfileRequiredServerThumbprint": "A server certificate thumbprint must be defined on the endpoint or a publish profile must be specified in the VSTS task.",
   "loc.messages.PublishProfileRequiredAppParams": "An application parameters file or a publish profile must be specified in the VSTS task.",
   "loc.messages.PublishProfileRequiredUpgrade": "Upgrade settings must be overridden or a publish profile must be specified in the VSTS task.",
+  "loc.messages.DefaultPortWarning": "The port '{0}' specified on your connection endpoint does not match the default ClientConnectionEndpoint of '19000'. This may have caused your failure to communicate with the cluster.",
```

Comment:
The message "This may have caused your failure to communicate with the cluster" seems hard to understand. Consider rewording.